tools/build-base-images.sh

```diff
@@ -21,5 +21,6 @@ set -ex
 
 HUB="${HUB:-istio.io/docker}"
 TAG="${TAG:?specify a tag}"
+PLATFORMS=linux/amd64,linux/arm64,linux/s390x,linux/ppc64le
 
-BUILDX_BAKE_EXTRA_OPTIONS="--no-cache --pull" DOCKER_TARGETS="docker.base docker.distroless docker.app_sidecar_base_debian_9 docker.app_sidecar_base_debian_10 docker.app_sidecar_base_ubuntu_xenial docker.app_sidecar_base_ubuntu_bionic docker.app_sidecar_base_ubuntu_focal docker.app_sidecar_base_centos_7 docker.app_sidecar_base_centos_8" make dockerx.pushx
+BUILDX_BAKE_EXTRA_OPTIONS="--no-cache --pull --set=base-default.platform=${PLATFORMS} --set=base-distroless.platform=${PLATFORMS}" DOCKER_TARGETS="docker.base docker.distroless docker.app_sidecar_base_debian_9 docker.app_sidecar_base_debian_10 docker.app_sidecar_base_ubuntu_xenial docker.app_sidecar_base_ubuntu_bionic docker.app_sidecar_base_ubuntu_focal docker.app_sidecar_base_centos_7 docker.app_sidecar_base_centos_8" make dockerx.pushx
```

Comment:
I think we actually want `base-default` and `distroless-default`. 

Basically the problem is we have two base images (base and distroless). Then we setup variants of all of our build images, and append -base or -distroless. But for some reason, we are also appending the variant to the base images, which doesn't seem right.

Reply:
Yeah. I make a mistake.  

I prefer #29200 did. so i closed this PR